drivers/syscon/Kconfig

```diff
@@ -0,0 +1,36 @@
+# SYSCON configuration options
+
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# SYSCON options
+#
+menuconfig SYSCON
+	bool "SYSCON drivers"
```

Comment:
Might be a good idea to do `SYSCON (System Controller) driver` for those who haven't heard of it before.

Reply:
ACK